src/js/index.js

```diff
@@ -0,0 +1,3 @@
+import { viewSingleListing } from './views/listings/index.js';
+
```

Comment:
This one you can do in the main index.js file.

Reply:
Thank you Chris!